### ISSUE TITLE
Prior Art Archive ImProvements Project Arplication (PrArArPrPrAr)

### ISSUE THREAD

**Issue by slifty** [BadIdeaFactory/corporate#96]
# I HEREBY INVOKE MY RIGHTS...
As described in section 11.11.B of the BIFFUD Corporate Bylaws I hereby invoke
my rights as a sentient being who has not uploaded their mind to the cloud for consideration of this project application by the BIFFUD Hive Mind.  With this
application I submit my interest in becoming a Member of BIFFUD and having this
project supported and adored by all who can 🤔.

## Project Information
- Project Name: The Prior Art Archive
- Project Haiku:

The past is prologue.
A prologue of bad ideas.
Let's store all of them!

- Project Analogy: It's like Battlestar Galactica for the United States Patent and Trademark Office

### Project Description

TL;DR :: Prior art stops patents from being issued: if there is prior art for a thing, than that thing cannot be patented.  This system will help streamline the process for discovering prior art.

Prior Art Archive (PrArAr) is an attempt to build out a system for collecting information about "prior art." -- this is a technical term in the legal field which I believe means "art that was created before this exact moment."

What we are planning to build is a network of high resolution scanners that can be mailed to communities around the world, asking them to scan ALL art created by children or adults.  For sculptures we will deploy a fleet of drones equipped with 3D cameras and / or giant claws in order to steal them and bring them back to our central archive.

PrArAr is a project of [The Underlay](https://underlay.mit.edu/) an edgy pop rock band founded at MIT in 2009.  In 2016 or whatever they pivoted to become `a global, distributed graph of public knowledge.`

Enough jibba jabba.  Here is what we put in the proposed scope of work:

```
Engineering, project management, and product management services for the PrArAr.  The goal is to build upon and generally improve the current PrArAr prototype in coordination with key stakeholders such as the US Patent and Trademark Office and in collaboration with Cisco and the Underlay Team.

Deliverables
  • New Upload Experience
    o Design and implementation of a new uploader experience (currently based on SFTP).The updated version will include:
      § A more natural and accessible flow and additionally provide a mechanism to upload through drag-and-drop on the website.
      § Metadata extraction triggered by the upload of new data.
    o Creation of an uploader / editor workflow that allows deleting, editing, and merging of documents and metadata.
  • Researcher (e.g. searching user) experience for viewing information related to a single document, single author, and single uploader.
  • Collaboration with Cisco engineers to help them maintain and potentially improve the search capabilities of the system.
  • Collaboration with other partners to improve entity extraction and application of patent IDs.
  • Updated user account page
    o Improving the way users view all of their uploaded documents by creating more accessible interfaces.
  • Where relevant, these efforts will also result in:
    o Creation of user needs documentation for USPTO, companies, law firms,and individuals
    o Creation of design specifications and wireframes
    o User testing and user feedback cycles
  • Create a system for identifying duplicate content and resolving those duplicates (e.g. via the merging interface referenced earlier)
  • General improvements to the system, as mutually identified over time

Responsibilities
Bad Idea Factory will be responsible for:
  • Conducting interviews and conversations with key stakeholders.
  • Designing, iterating on, and implementing the frontend (and supporting backend) of PrArAr.
  • Communicating progress and regularly pushing code and design progress to GitHub.
  • Performing deployment of the web application to staging and production servers.
  • Managing the project by regularly establishing priorities and processes in order to mesh together various contributing parties.

Bad Idea Factory will rely on the Underlay team for:
  •Providing regular feedback on design and implementation progress.
  •Assisting in coordination with external stakeholders.
  •Providing access to necessary computational resources (e.g. staging and production servers, code bases, and necessary backends / APIs to power the system)
```

### How is this project a bad idea?
One of our owners is an MIT employee, and since MIT has a conflict of interest policy this means that we would need to make sure that that owner sees absolutely no benefit from this relationship.  In order to ensure this, we will need to go out of our way to make sure that the project creates net harm for Bad Idea Factory.

For example, because of this conflict of interest MIT would require us to not collect the 2% BIF tithe since that would ultimately benefit the company.

### If this project were a D&D Character, what alignment would it be and why?
Alignment: Neutral Good

### Where are the lulz?

* Adoption of this project would represent a legal commitment to sabotage ourselves.
* Yet another highly respected organization would be hiring a company called Bad Idea Factory, which is a concept that will probably never get old
* We get to index ALL BAD IDEAS IN THE KNOWN UNIVERSE.
* We will be in a great position to be able to upload our OWN bad ideas into the database.

### How does this project make people thinking face emoji?
This system will help more people get to 🤔 and create things without getting sued by patent trolls, because it will help the USPTO avoid issuing bad patents.

### Who is involved?
- Name: Dan Schultz
- Twitter: @slifty
- Github ID: slifty
- Skillz: Project management, infiltration, 
- Project role / expectations: Project lead; programming; talking to people all the time
- Project stake: 100%

- Name: Justin Reese
- Skillz: UX design / full stack development
- Project role / expectations: UX design / full stack development
- Project stake: 0% (paid contractor)

### Who will be the project's Comptroller?
Dan Schultz

### Is this realistic to implement via BIFFUD?
MONEY MAKES ANYTHING REALISTIC TO IMPLEMENT RIGHT?

### Money Stuff
This project is FUNDED.
As noted earlier, in order to sign the contract we would have to waive the BIF Tithe (2%) for money paid by MIT.  *Cough*

## Next Steps
1. Attend the next scheduled BIFFUD plotting session to plead your case.
2. [List out your project plan here, in a series of steps]

### How (often) will you be providing updates to the organization?
[Ideas are great, action is better!  How often should we expect you
to be able to make progress? How would you like to report back]
